README.md

```diff
@@ -37,4 +37,4 @@ than by hand. For example, have a little Flutter program that
 generates the PNGs so that we can easily re-run the program and
 regenerate the images if we want to e.g. change the resolution or
 adjust the text slightly. For an example,
-see [tile_mode.dart](dart-ui/tile_mode.dart).
+see [tile_mode.dart](src/dart-ui/tile_mode.dart).
```

Comment:
you should update this with instructions on how to do it automatically

Reply:
as in, how to use the script you wrote

Reply:
Actually, I did write a README.md with that info, I just put it in the src subdir.  I'll move it here.